### DISCUSSION TITLE
Blacklist Servers (Imgur & Redgifs)

### DISCUSSION THREAD

**Discussion by rlgurney** [MalloyDelacroix/DownloaderForReddit#358] [Q&A]
How would one go about blocking any and all posts that link to A) dead sites and/or B) sites that have blocked NSFW content since the original Reddit post was made?

Specifically, I want to prevent Downloader for Reddit from trying to pull images from Imgur or Redgifs, as they will never return anything other than an error at this point.

Thanks!

**Comment by MalloyDelacroix** [comment]
This feature has been added in v3.16.0.  There is now a list on the core settings page that displays each available extractor with a checkbox next to it.  Uncheck the extractor and it will not be used during download.

Note that the direct link extractor will download from any site as long as the URL ends in an extension.  This may still download from sites you attempt to blacklist if the post was a direct link.

If it is a site that hosts videos, you should also check the sites listed in the "Video Sites" section of the settings and uncheck the site in that list too.  This list controls a different extractor, and it is possible to still get downloads if left enabled here.
